client/verta/docs/tutorials/deployment/endpoint_update_config.rst

```diff
@@ -0,0 +1,57 @@
+Updating Endpoint with Config File
+==================================
+
+.. TODO: link to Endpoint.update tutorial
+
+Sometimes, it is more convenient to specify update configuration in a separate, human-readable file, than to cram everything in the Python script or the CLI command.
+
+For example:
+
+.. code-block:: json
+
+        {
+            "run_id": <experiment run id>,
+            "strategy": "canary",
+            "canary_strategy": {
+                "progress_step": 0.05,
+                "progress_interval_seconds": 30,
+                "rules": [
+                    {"rule": "latency_avg_max",
+                     "rule_parameters": [
+                         {"name": "threshold",
+                          "value": "0.1"}
+                    ]},
+                    {"rule": "error_4xx_rate",
+                     "rule_parameters": [
+                        {"name": "threshold",
+                         "value": "1"}
+                    ]}
+                ]
+            }
+        }
```

Comment:
This is a mixed formatted JSON, it seems. I'd recommend using a standard formatting instead, as this is pretty unusual and harder to consume.

Reply:
Hmm see if it looks better. Also the <experiment run id> part breaks the code block. I have to put it inside a string (i.e double quotes) so that highlighting is done properly.

Reply:
Yeah, makes sense to require string